src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -51,6 +55,8 @@
     private final List<List<JSONPointer>> additionalIdentifiers = new ArrayList<>();
     private final List<JSONPointer> readOnlyProperties = new ArrayList<>();
     private final List<JSONPointer> writeOnlyProperties = new ArrayList<>();
+    @Getter(AccessLevel.NONE)
```

Comment:
Let's remove the `@Getter` here

Reply:
this suppresses the getter so someone can't do something like:

```
Map<String, Handlers> handlerMap = schema.getHandlers();
handlerMap.put("create", new Handler());
```

Setting the access level to none overrides the @Getter at the top of the class:
https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/blob/1e99ac138807111f16c5368cf6977ec09dbd6d42/src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java#L35-L39

Reply:
Got it.